### PR TITLE
debuggability for docker issues on travis

### PR DESCRIPTION
**Pull request by mparkhe**
## What changes are proposed in this pull request?
 
debuggability for docker issues on travis
 
## How is this patch tested?
 
(Details)
 
### Is this a user-facing change? 

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.
 
(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
 
### What component(s) does this PR affect?
 
- [ ] UI
- [ ] CLI 
- [ ] API 
- [ ] REST-API 
- [ ] Examples 
- [ ] Docs
- [ ] Tracking
- [ ] Projects 
- [ ] Artifacts 
- [ ] Models 
- [ ] Scoring 
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:
 
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: tests/projects/utils.py [hunk 1]
```diff
@@ -39,9 +40,16 @@ def assert_dirs_equal(expected, actual):
...
+    try:
+        client.images.build(tag='mlflow-docker-example', forcerm=True,
+                            dockerfile='Dockerfile', path=TEST_DOCKER_PROJECT_DIR)
+    except BuildError as build_error:
+        for chunk in build_error.build_log:
+            print(chunk)
+        raise build_error
+    except APIError as api_error:
+        print(api_error.explanation)
+        raise api_error
```
Comment (sueann) [inline tests/projects/utils.py]: could there be other errors?

**Review by sueann — APPROVED** [review]
one question - but i'm good with merging ASAP so we can figure this out if needed. though the travis run for this should show the info?

File: tests/projects/utils.py (hunk 1 above)
Comment (mparkhe) [reply tests/projects/utils.py]: i says the following are raised. `TypeError` would be just retrown

        Raises:
            :py:class:`docker.errors.BuildError`
                If there is an error during the build.
            :py:class:`docker.errors.APIError`
                If the server returns any other error.
            ``TypeError``
                If neither ``path`` nor ``fileobj`` is specified.

**Comment by mparkhe** [comment]
> though the travis run for this should show the info?

one advantage of a test failing on travis ;-)

**Pull request merged**